### ISSUE TITLE
Configuring the amount of consistency reads

### ISSUE THREAD

**Issue by 2tefan** [kubernetes/utils#293]
**Is your feature request related to a problem? Please describe.**

Currently, the consistency reads are hard-coded to 3:
https://github.com/kubernetes/utils/blob/3b25d923346b3814e0898684c97390092f31a61e/mount/mount_helper_unix.go#L36

This gives three retries to read the file and if they are equal it succeeds.
https://github.com/kubernetes/utils/blob/3b25d923346b3814e0898684c97390092f31a61e/io/read.go#L43-L63

**Describe the solution you'd like in detail**

It would be nice to have an option to configure this setting, but I am not sure how.

**Additional context**

Maybe a bit of topic, but is this test even useful? Because it makes sense to test if the file changed. But if this is used to check if something else is currently getting mounted, before starting their own mount, is this even relevant? Because from the time this check passes until the actual mount happens, it is also possible that a mount happens in the meantime.

Or is there something I am missing?

**Comment by tamilselvan1102** [comment]
Since it is a utility file and it can be used by various package and repo, the constant setting should be in the package.
it is not good practice ask some setting/config before using this utils.

**Comment by tamilselvan1102** [comment]
if you stratified, please close the issue.
